Cabal/src/Distribution/Verbosity.hs

```diff
@@ -98,23 +98,24 @@ instance Bounded Verbosity where
 instance Binary Verbosity
 instance Structured Verbosity
 
--- We shouldn't print /anything/ unless an error occurs in silent mode
+-- | In 'silent' mode, we should not print /anything/ unless an error occurs.
```

Comment:
OOI, what's the meaning of `/` in a haddock?

Reply:
it's italics

Reply:
oh, cool